From f0b325a630277120e67bc6432e99901f96621f88 Mon Sep 17 00:00:00 2001
From: Tom Rix <trix@redhat.com>
Date: Sun, 12 May 2024 11:21:39 -0600
Subject: [PATCH] tensile workaround cache problem

---
 Tensile/Common.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Tensile/Common.py b/Tensile/Common.py
index 6ececf1c47b3..672cef5aa842 100644
--- a/Tensile/Common.py
+++ b/Tensile/Common.py
@@ -2037,8 +2037,8 @@ def GetAsmCaps(isaVersion):
       ignoreCacheCheck = True
 
     # check if derived caps matches asm cap cache
-    if not ignoreCacheCheck and derivedAsmCaps != CACHED_ASM_CAPS[isaVersion]:
-      printExit("Cached asm caps differ from derived asm caps for {}".format(isaVersion))      
+    # if not ignoreCacheCheck and derivedAsmCaps != CACHED_ASM_CAPS[isaVersion]:
+    #  printExit("Cached asm caps differ from derived asm caps for {}".format(isaVersion))
     return derivedAsmCaps
   else:
     printWarning("Assembler not present, asm caps loaded from cache are unverified")
-- 
2.45.0

